account_invoice_line_description/views/res_config_view.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <record id="view_account_config" model="ir.ui.view">
+        <field name="name">account settings</field>
+        <field name="model">account.config.settings</field>
+        <field name="inherit_id" ref="account.view_account_config_settings"/>
+        <field name="arch" type="xml">
+            <xpath
+                expr="//label[@for='module_account_budget']"
```

Comment:
This field is only visible if user belongs to `account.group_account_user` (aka Accountant) group.
For this reason I see two possibilities:

1. add this info in the README file
2. move `div` after https://github.com/odoo/odoo/blob/10.0/addons/account/views/res_config_view.xml#L104

What do you think?